src/Result/AddressList.php

```diff
@@ -6,7 +6,18 @@
 
 class AddressList extends \BTCPayServer\Result\AbstractListResult
 {
+    public function all(): array
+    {
+        $r = [];
+        foreach ($this->getData()['addresses'] as $addressData) {
+            $r[] = new \BTCPayServer\Result\Address($addressData);
+        }
+        return $r;
+    }
+
     /**
+     * DEPRECATED - PLEASE USE all()
```

Comment:
How about a message like this (idem for all other changes like this one):

```
DEPRECATED: Please use `all()`

TODO: Remove during next backwards compatibility break.
```

Reply:
Agreed. Added to PR.